pkg/types/azure/machinepool.go

```diff
@@ -3,6 +3,9 @@ package azure
 // MachinePool stores the configuration for a machine pool installed
 // on Azure.
 type MachinePool struct {
+	// Zones is list of availability zones that can be used.
```

Comment:
is there some static validation that can be performed like https://github.com/openshift/installer/blob/a969950ec2a63eab8b0bb8017a3b489211bbae00/pkg/types/aws/validation/machinepool.go#L15-L19 ??